osquery/logger/plugins/windows_event_log.h

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+// clang-format off
+#include <windows.h>
```

Comment:
For consistency would you mind adding on the `#define WIN32_LEAN_AND_MEAN` or are you relying on extra libs? We'll be moving this logic to CMake eventually, but just to ensure it's not pulling in any additional libs.